### ISSUE TITLE
A very weird bug of tf_debug: Non-Ok-status env->NewWritableFile(file_path, &f) status: Resource exaustated

### ISSUE THREAD

**Issue by rylanchiu** [tensorflow/tensorflow#11628]
### System information
- **Have I written custom code**:  Yes
- **OS Platform and Distribution**: CentOS
- **TensorFlow installed from (source or binary)**: binary
- **TensorFlow version**: 1.2
- **Python version**:  3.4

When I run the seq2seq model, I got the `nan` loss. Thus I use `tf_debug` to find out where the problem occurs. I use `tf_debug` by 
 
    sv = tf.train.Supervisor(logdir=FLAGS.log_root,
                             is_chief=True,
                             saver=saver,
                             summary_op=None,
                             save_summaries_secs=60,
                             save_model_secs=FLAGS.checkpoint_secs)
    sess = sv.prepare_or_wait_for_session(config=tf.ConfigProto(
        allow_soft_placement=True))
    sess = tf_debug.LocalCLIDebugWrapperSession(sess)
    sess.add_tensor_filter("has_inf_or_nan", tf_debug.has_inf_or_nan)

But it got such logs and exit:

    tfdebug Non-OK-status: env->NewWritableFile(file_path, &f) status: Resource exaustated: /tmp/tfdbg_9_gcc3sc/gradients/output/Reshape_1_grad/Reshape_0_DebugIdentity_150023213
    Aborted (core dumped)

I think it is kind of issue related to tf_debug and it may be new, since I can not find anything when I google the error.

**Comment by asimshankar** [comment]
This error occurs when writing to the filesystem fails - for example if there is no space left on the device, or if the disk quota exceeded or if there were too many files etc. (see [`errors.cc`](https://github.com/tensorflow/tensorflow/blob/r1.2/tensorflow/core/platform/posix/error.cc#L86))

Unfortunately, the error message given doesn't give more detail (I'll look into fixing that), but the underlying problem does appear to be that you're hitting some sort of space or user limit issues when writing to `/tmp/tfdbg_9_gcc3sc/gradients/output/Reshape_1_grad/Reshape_0_DebugIdentity_150023213`

Could you look into how much space is left on the device and whether you're running into disk limits?

**Comment by rylanchiu** [comment]
@asimshankar  Thanks for your reply! But it is unlikely that no space left on the device. Cause I run the code on a server, whose space is much more than enough.  Do you think user authentication can be a possible reason. But I have no problem on writing under my path.

**Comment by asimshankar** [comment]
I didn't quite follow what you meant by "I have no problem on writing under my path". Could you elaborate on that?

But regardless, it seems like the `fopen` call [here](https://github.com/tensorflow/tensorflow/blob/r1.2/tensorflow/core/platform/posix/posix_file_system.cc#L150) that is failing, so it does seem to be the filesystem rejecting the open call. So perhaps there are some `ulimits` in place?

**Comment by tensorflowbutler** [comment]
This issue is automatically closed due to lack of activity. Please re-open if this is still an issue for you. Thanks!
